rse-ssh.Rmd

```diff
@@ -0,0 +1,449 @@
+# Working Remotely {#rse-ssh}
+
+```{r rse-ssh-setup, include=FALSE}
+source("_common.R")
+```
+
+Let's take a closer look at what happens when we use the shell
+on a desktop or laptop computer.
+The first step is to log in
+so that the operating system knows who we are and what we're allowed to do.
+We do this by typing our username and password;
+the operating system checks those values against its records,
+and if they match,
+runs a shell for us.
+
+As we type commands,
+the 1's and 0's that represent the characters we're typing are sent from the keyboard to the shell.
+The shell displays those characters on the screen to represent what we type,
+and then,
+if what we typed was a command,
+the shell executes it and displays its output (if any).
```

Comment:
Not sure this is necessary. It's nice "interesting to know", but doesn't relate to the learning outcomes.

Reply:
Shortened - let me know?